src/components/Molecules/VideoBanner/VideoBanner.js

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const Video = styled.video`
+  width: 100%;
+  background-image: url(${({ poster }) => poster});
+  background-size: 100% 100%;
+  object-fit: contain;
+  overflow: hidden;
+`;
+
+const VideoBanner = ({ video, autoPlay, loop }) => {
+  const videoEl = useRef(null);
+
+  const onPlay = () => {
+    videoEl.current.play();
+  };
+
+  useEffect(() => {
+    onPlay();
+  });
+
+  return (
+    <Video ref={videoEl} autoPlay={autoPlay} loop={loop} muted playsinline>
```

Comment:
I guess it doesn't _really_ matter here (as we can "hard set" it within the Contentful integration), but we probably need to ensure these bg videos **don't** loop; it was part of an accessibility update following a DAC audit and we don't wanna give CMS options to something that'll have a negative impact.

Reply:
Also, don't we need to pass the Poster attr?

Reply:
Yes, height we can be defined in gatsby app. Ok I will make sure loop is always false.

Adding back poster `attr`! I tried to use bg image instead because poster wasn't filling the container. I want to test it in real use to see how it will look.